code/modules/admin/verbs/smite.dm

```diff
@@ -0,0 +1,69 @@
+/obj/effect/lightning
+	name = "divine retribution"
+	icon = 'icons/effects/224x224.dmi'
+	icon_state = null
+	pixel_y = -32
+	pixel_x = -100
+	layer = 16
+
+	proc/start(atom/location)
+		loc = get_turf(location)
+		flick("lightning",src)
+		playsound(src,'sound/effects/thunder.ogg',50,1)
+		spawn(20)
+			qdel(src)
+
+/client/proc/cmd_smite(mob/living/M in mob_list)
+	set category = "Fun"
+	set name = "Smite"
+	if(!holder)
+		usr << "no"
+		return
+
+	var/options = list("Brute","Burn","Toxic","Oxygen","Clone","Brain","Stamina","Heal","Gib","Cancel")
+	var/requiresdam = options - list("Heal","Gib","Cancel")
+
+	var/confirm = null
+	var/damtype_word = null
+	var/dam = 0
+
+	confirm = input(src, "Really smite [M.name]([M.ckey])?", "Divine Retribution") in list("Yeah", "Nah")
+	if(confirm == "Nah")
+		return
+	// Changed so that the damtype isn't used where it shouldn't be used. damtype_word is the word... damtype is an int.
+	damtype_word = input(src, "What kind of damage?", "PUT YOUR FAITH IN THE LIGHT") in options
+	if(damtype_word in requiresdam)
+		dam = input(src, "How much damage?", "THE LIGHT SHALL BURN YOU") as num
+		if(dam == 0)
+			damtype_word = "Cancel"
+	if(damtype_word == "Cancel")
```

Comment:
I'm assuming you mean to remove lines 38 and 39 right?
Well if I did that then it would break the cancel option as only the only the options that do a numbered amount of damage asks for how much damage then checks if the damage entered is 0 which then cancels the smiting. So it would probably just do the animation and sound an then nothing but log something about smiting with cancel. I could move the check for 0 damage out of the scope where it ask for how much damage but then it would break gib and heal because they don't use the dam variable so it would be 0 always. I could make line 38 a return though so it stops faster.